src/cr/cube/vector.py

```diff
@@ -239,10 +231,23 @@ def anchor(self):
         return self._base_vector.anchor
 
 
-class _BasePrunedOrHiddenVector(_BaseTransformationVector):
-    def __init__(self, base_vector, opposite_vectors):
+class PrunedVector(_BaseTransformationVector):
+    def __init__(self, base_vector, opposite_vectors, prune=True):
```

Comment:
Default parameters are best avoided except in public (user) interface classes where they can improve ease-of-use (at the expense of code complexity). In particular, they should not be used just to avoid updating a set of unit tests.  This parameter is always provided on all existing calls. Let's leave it as `, prune` and ditch the `=True`.